### DISCUSSION TITLE
Place informational node without overlapping any nodes in the paper

### DISCUSSION THREAD

**Discussion by DarkMagus** [clientIO/joint#2771] [Q&A]
### Introduction

**The user case**: 
We have nodes that are placed on the paper. When the user interacts with a node we would like to display a new node that contains specific information about the node. 
*The question is the following*: 
is there any JointJS utilities that allows us to place this specific information about a node in the paper with the condition that the informational node should not overlap with any other nodes?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You might find this example helpful: https://www.jointjs.com/demos/text-position-based-on-space-availability
It implements the logic to find the vacant space for the label.
